### ISSUE TITLE
Pose Estimation

### ISSUE THREAD

**Issue by ansabsheikh9** [libornovax/master_thesis_code#51]
Dear Mr. Novak,
                           can you please comment on how are you evaluating pose estimation in you work?
Best Regards
Ansab

**Comment by libornovax** [comment]
Hopefully I will be able to publish a new technical report soon, where I introduce a new version and have a bit more details about pose estimation.

For now I can refer you to the [KITTI evaluation results](http://www.cvlibs.net/datasets/kitti/eval_object_detail.php?&result=8bbefcbd45337982ea83f19f606df7de71f23166), where I compete with the new version of the algorithm in 2D and 3D.

**Comment by libornovax** [comment]
The current status is that I unfortunately I did not manage to publish the new technical report :(

However, to answer the question, in the master thesis the evaluations I made were based on mean distance error as described in *Arsalan Mousavian, Dragomir Anguelov, John Flynn, and Jana Kosecka. 3d bounding box estimation using deep learning and geometry.* and then I also checked the KITTI results, where the difference between 2D and 2D+rotation can tell you how well the pose is being estimated. In my case the results are basically the same, which suggests that the rotation estimation is quite good.

About the 3D bounding box evaluation in KITTI: This evaluation did not exist at the time of publishing the thesis so I only added the results later, however from a new version of the detector, where the 3D box is regressed directly, therefore improving the 3D precision of the boxes significantly. The original version as published in the thesis would probably score almost nothing as they require 70% IoU for cars, which in the case of 3D is very strict.

I know it is not much information, but hope it helps a bit.
